packages/python3.rb

```diff
@@ -3,33 +3,48 @@
 class Python3 < Package
   description 'Python is a programming language that lets you work quickly and integrate systems more effectively.'
   homepage 'https://www.python.org/'
-  @_ver = '3.9.2'
+  @_ver = '3.9.4'
   version @_ver
   license 'PSF-2.0'
   compatibility 'all'
-  source_url "https://www.python.org/ftp/python/#{@_ver}/Python-#{@_ver}.tar.xz"
-  source_sha256 '3c2034c54f811448f516668dce09d24008a0716c3a794dd8639b5388cbde247d'
+  source_url "https://www.python.org/ftp/python/#{@_ver}/Python-#{@_ver}.tgz"
+  source_sha256 '66c4de16daa74a825cf9da9ddae1fe020b72c3854b73b1762011cc33f9e4592f'
 
   binary_url({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/python3-3.9.2-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/python3-3.9.2-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/python3-3.9.2-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/python3-3.9.2-chromeos-x86_64.tar.xz'
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/python3-3.9.4-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/python3-3.9.4-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/python3-3.9.4-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/python3-3.9.4-chromeos-x86_64.tar.xz'
   })
   binary_sha256({
-    aarch64: '83b292381a6fcda63f3c8c6c4b682694be673b42ed14eb1482e5cfe6f1c0f835',
-     armv7l: '83b292381a6fcda63f3c8c6c4b682694be673b42ed14eb1482e5cfe6f1c0f835',
-       i686: '8f3ba22ea8824a958aafe9127e8fa078406323a6d9785e2986d3dc4ac45f671a',
-     x86_64: 'd3195034ad9d26e9e4c0c08c35bff6f551cc5661b970284c96deb81f32d34fb8'
+    aarch64: 'fe4fde3f9738ac76096dfdc13ff6931543e2f612035421e294f87d2bfc09773c',
+     armv7l: 'fe4fde3f9738ac76096dfdc13ff6931543e2f612035421e294f87d2bfc09773c',
+       i686: 'a9e7b61911679cda55f65cf5ca29076ee2ac4540c8dd7bda5322df66d4d6e4bd',
+     x86_64: 'cd3a63250e76cd887a620df69f37d349d4cea0d39e3d8bc2ceef9400a40befbb'
   })
 
-  depends_on 'bz2'
-  depends_on 'xzutils'
-  depends_on 'sqlite'
-  depends_on 'krb5'
-  depends_on 'libtirpc'
-  depends_on 'tk' => :build # yes this creates a circular dependency.
+  depends_on 'bz2' => :build
+  depends_on 'fontconfig'
+  depends_on 'freetype'
   depends_on 'gdbm' => :build
+  depends_on 'glib'
+  depends_on 'graphite'
+  depends_on 'harfbuzz'
+  depends_on 'krb5' => :build
+  depends_on 'libdb'
+  depends_on 'libpng'
+  depends_on 'libtirpc' => :build
+  depends_on 'libx11'
+  depends_on 'libxau'
+  depends_on 'libxcb'
+  depends_on 'libxext'
+  depends_on 'libxft'
+  depends_on 'libxrender'
+  depends_on 'libxss'
+  depends_on 'sqlite' => :build
+  depends_on 'tcl'
+  depends_on 'tk'
+  depends_on 'xzutils' => :build
```

Comment:
Does this expand the list of core dependencies?  In other words, does `tools/core_packages.txt` need updated?

Reply:
Tcl & tk aren't in core... But does it matter?

What changes for installs if we don't add it to core?

Nothing?

Tcl/tk will also get installed with just adding the deps to the python3 package file anyways.

Reply:
Yes, it matters.  The list of core packages should be complete.  Since python3 is in core, the dependencies should be included in core also.

Reply:
Added tcl & tk to core packages in this PR.

Reply:
Dependency list is not complete.

Reply:
Removing the tkinter library (e.g. `/usr/local/lib/python3.9/lib-dynload/_tkinter.cpython-39-i386-linux-gnu.so`)  from depenendency analysis removes all the other x11 dependencies.

So I'm just removing that file from consideration for dependencies, and now there are no new core dependencies.